sycl/plugins/level_zero/CMakeLists.txt

```
@@ -99,6 +99,11 @@ add_sycl_plugin(level_zero
     ${XPTI_LIBS}
 )
 
+if (WIN32)
+ # 0x800: Search for the DLL only in the System32 folder
+ target_link_options(pi_level_zero PUBLIC /DEPENDENTLOADFLAG:0x800)
```

this file uses two spaces for indentation
```suggestion
  # 0x800: Search for the DLL only in the System32 folder
  target_link_options(pi_level_zero PUBLIC /DEPENDENTLOADFLAG:0x800)
```